corehq/db/config.yml

```diff
@@ -0,0 +1,20 @@
+# HQ database configuration for multiple database. This gets mapped to Django's DATABASES dict
```

Comment:
could an in-between solution be something like an HQSettingsManager class which is an intermediary between the yaml file and `settings.py`? 

In the so example, I'm thinking rather than calling `parser.get('x')`, you call `settings_manager.get_variable('x')`, but then you also have special functions like `settings_manager.get_databases()` or `settings_manager.get_proxy_database()`, etc.